release-content/0.15/release-notes/13454_vbao.md

```diff
@@ -0,0 +1,13 @@
+**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:
```

Comment:
```suggestion
**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique and is particularly important on thin geometry (like the chair legs in the scene below):
```